servers/lib/test/files.e2e-spec.ts

```diff
@@ -0,0 +1,61 @@
+
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from '../src/app.module';
+import { FilesModule } from '../src/files/files.module';
+import { GraphQLModule } from '@nestjs/graphql';
+import { join } from 'path';
+import { FilesResolver} from '../src/files/files.resolver';
+import { FilesService } from '../src/files/files.service';
+import { ApolloDriver } from '@nestjs/apollo';
+import { ConfigService } from '@nestjs/config';
+
+describe('FilesResolver (e2e)', () => {
+  let app: INestApplication;
+  let filesService: FilesService;
+
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [
+        GraphQLModule.forRoot({  
+            driver: ApolloDriver,
+            autoSchemaFile: join(process.cwd(), 'src/schema.gql'),
+            debug: false,
+            playground: true,
+        }),
+        FilesModule],
+        providers: [FilesResolver, FilesService, ConfigService]
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+
+    filesService = moduleFixture.get<FilesService>(FilesService);
+  });
+
+    it('should return an array of file names for a given path', async() => {
```

Comment:
This is a unit test. Please rename the file to indicate that it's an integration test but not end2end.
An end2end test requires running the complete app with "yarn start:test" and then running tests using graphql queries sent from the test code to the live microservice.

Reply:
Doing this end2end test would take the integration test coverage to near 100%.

Reply:
I have just pushed the update of app.e2e-spec.ts